code/modules/fish/fish_items.dm

```diff
@@ -214,3 +232,37 @@ var/list/fish_items_list = list("goldfish" = /obj/item/weapon/fish/goldfish,
 		qdel(src)
 		return TRUE
 	..()
+
+/obj/item/weapon/lobster/attackby(var/obj/item/O, var/mob/user) // extracting tail and claw meat from a sea cockroach
+	if(iswirecutter(O))
+		to_chat(user, "<span class='notice'>You crack open the shell of \the [src] and pull out the claw meat while separating the tail!")
+		new /obj/item/weapon/reagent_containers/food/snacks/raw_lobster_meat(get_turf(src))
+		new /obj/item/weapon/reagent_containers/food/snacks/raw_lobster_meat(get_turf(src))
+		new /obj/item/weapon/reagent_containers/food/snacks/raw_lobster_tail(get_turf(src))
+		qdel(src)
+		return
+	..()
+
+/obj/item/weapon/reagent_containers/food/snacks/raw_lobster_tail/attackby(var/obj/item/O, var/mob/user) // extracting the meat from the tail, just makes normal lobster meat
+	if(iswirecutter(O))
+		to_chat(user, "<span class='notice'>You crack open the remains of the shell from \the [src] and pull out the meat!")
+		new /obj/item/weapon/reagent_containers/food/snacks/raw_lobster_meat(get_turf(src))
+		qdel(src)
+		return
+	..()
+
+/obj/item/weapon/steamed_deluxe_lobster/attackby(var/obj/item/O, var/mob/user) // cracking the shell of a steamed lobstroso
```

Comment:
from a readability perspective it seems bad to keep the attackbys for these far away from the definition of each object, keep them together